packages/manager/src/features/CloudView/Dashboard/CloudPulseDashboardLanding.tsx

```diff
@@ -0,0 +1,149 @@
+import { Dashboard } from '@linode/api-v4';
+import React from 'react';
+
+import CloudViewIcon from 'src/assets/icons/Monitor.svg';
+import { CircleProgress } from 'src/components/CircleProgress';
+import { Paper } from 'src/components/Paper';
+import { StyledPlaceholder } from 'src/features/StackScripts/StackScriptBase/StackScriptBase.styles';
+
+import { GlobalFilters } from '../Overview/GlobalFilters';
+import { FILTER_CONFIG } from '../Utils/FilterConfig';
+import { getUserPreference } from '../Utils/UserPreference';
+import { CloudPulseWidgetFilters } from '../Widget/CloudViewWidget';
+import { CloudPulseDashboard } from './Dashboard';
+
+export const CloudPulseDashboardLanding = () => {
+  const [filterValue, setFilterValue] = React.useState<{ [key: string]: any }>(
+    {}
+  );
+
+  const [dashboard, setDashboard] = React.useState<Dashboard>();
+  const [isPrefLoaded, setIsPrefLoaded] = React.useState<boolean>(false);
+
+  const filterReference: { [key: string]: any } = React.useRef({});
+
+  const handleGlobalFilterChange = React.useCallback(
+    (filterKey: string, value: any) => {
+      filterReference.current[filterKey] = value;
+      setFilterValue({ ...filterReference.current });
+    },
+    []
+  );
+
+  const dashboardChange = React.useCallback((selectedDashboard: Dashboard) => {
+    setDashboard(selectedDashboard);
+  }, []);
+
+  const checkIfAllMandatoryFiltersAreSelected = () => {
+    if (FILTER_CONFIG && FILTER_CONFIG.get(dashboard!.service_type!)) {
+      const serviceTypeConfig = FILTER_CONFIG.get(dashboard!.service_type!);
+
+      for (let i = 0; i < serviceTypeConfig!.filters.length; i++) {
+        if (
+          filterValue[serviceTypeConfig!.filters[i].configuration.filterKey] ==
```

Comment:
if no key contains boolean value in filterValue object then we can use 
```suggestion
          !Boolean(filterValue[serviceTypeConfig!.filters[i].configuration.filterKey])
```
instead of undefined check